lib/TextOperations.cpp

```diff
@@ -210,4 +248,28 @@ std::string TextOperations::escapeString(std::string input)
 	return input;
 }
 
+int TextOperations::extractNumberAfterHash(const std::string & input)
+{
+    size_t hashPos = input.find('#');

```

Comment:
Use tabs instead of spaces

Reply:
Done. I made copy past mistake, that was not caught because my VS automatic tools are off.
If not everybody sticks with code formatting guide, VS will change everything and then will let you wonder what are changes and what is whitespaces.